portafly/src/i18n/locales/en/accounts.yml

```diff
@@ -1,20 +1,107 @@
+  role_header_value: #----PROPOSED
+    admin_you: Admin (you)
+    admin: Admin
+    member: Member
+  role_header_value_description: Container with possible values of the roles of each team member
+  permission_groups_header: Permission groups
+  permission_groups_header_description: Column listing the permisions mapped to each team member
+
+## Tab: Invitations
+## Toolbar
+#-------# Bulk selector: available in shared.yml
+#-------# Delete button: available in shared.yml
+invitations_filter:  #----PROPOSED
+ by_recipient: Recipient
+ by_acceptance: Acceptance
+invitations_filter_description: List containing the criteria to filter invitations
+invitations_typeahead:
+  filter_by_recipient: Filter by recipient
+  filter_by_acceptance: Filter by acceptance
+invitations_typeahead_description: Default text to show when the user filters by recipient or acceptance
+
+invitations_table:
+  aria_label: Invitations table
+  aria_label_description: Aria label of the table of invitations
+  empty_state: Currently, there are no invitations sent. Start sending invitations to your team members. #----PROPOSED
+  empty_state_description: Message to show when there are no invitations to show, and prompt the user to send some invites
+  recipient_header: Recipient
+  recipient_header_description: Column listing the email addresses of invitation recipients
+  sent_header: Sent on UTC time #----PROPOSED
+  sent_header_description: Column listing the date when invitations were sent
+  acceptance_header: Accepted on UTC time #----PROPOSED
+  acceptance_header_description: Column listing the date when the invitations were accepted
+  acceptance_values: #----PROPOSED
+    yes_with_time: Yes - {{utc_time}}
+    not_yet: Not yet
+  acceptance_values_description: Container with the possible responses to whether the invitation has been accepted or not.
+  actions_header: Actions
+  actions_header_description: Column listing the available actions to perform with each invitation
+
+# Modal: Invitation deletion
+invitation_deletion:
+  page: Delete selected invitations #----PROPOSED
+  page_description: Body title of modal dialog to delete the selected invitations from the table
+  admonition: This action will irreversibly delete the selected invitations. Please confirm below to proceed with the deletion. #----PROPOSED
+  admonition_description: Admonition text to explain that the removal of the selected invitations cannot be undone.
+  #-------# Delete button: available in shared.yml
+  #-------# Cancel button: available in shared.yml
+
+# Modal: Invitation to a new member
+invitation_sending:
+  page: Invite a new team member
+  page_description: Body title of modal dialog to send invitation to a new team member
+  label: Send invitation to *
+  label_description: Label prompting the user to specify the email address of the invitation recipient
+  validation: Please indicate a valid email address of the colleague or friend you want to invite to your team. #----PROPOSED
```

Comment:
@porueesq I think that for technical reasons it's better to use "Account" here. The user is inviting somebody to use their same Account, and Account is a concept in 3scale, team is not. I understand the confusion, which makes me think that a paragraph above the form explaining what's the output of this action would be beneficial. @thomasmaas what do you think?

Reply:
@cntlsn If the title of the page is **Team members**, but the concept refers to a group of people sharing the same account, we'll need to add a paragraph to explain this.

Also, we'll need to find a way to distinguish from an _user account_ (the one used to login to 3scale) versus an _account_ defined from a technical perspective. Let's avoid semantic overload.

Reply:
@porueesq I agree with adding a paragraph in the page header to explain this!
About the differentiation between Account (the 3scale entity) and User account (linked to a specific user), I agree that this is misleading, I'm just not sure this can be solved in this context.
Here is my thinking: users are familiar with the concept of a User account (because that is everywhere on the internet), but they are not with the concept of an Account as used by 3scale, which by the way is linked to the Group/Organization, not to a specific user (unless only one user exists). So to make things easy for users, I would keep the concept of a User account (or simple "Account") intact, and change the wording for the 3scale Account to something like Organization or Team, which is actually a collection of user accounts at the Company level.
But, it is possible that this would be a breaking change, so I'm not sure we want to make this change at this point...
@thomasmaas comments are welcome.

Reply:
I would probably use 'organization' and reserve 'teams' for when we might one day introduce something that lets you further organize your org into teams.

Reply:
@thomasmaas @porueesq 
So, to summarize and make sure we are all on the same page.
* What is now a 3scale "Account" concept, will become an "Organization":
  * There will always be two types of Organizations (one API Provider organization, multiple API Consumer organizations) and will both be called with the same "Organization" term
* Users of the Organizations will be simply called "Users", and all actions related to them will refer to them as such (eg. "Add user", "Delete user")
* Current logged-in Admin user of the API Provider organization will be able to:
  * manage its organization
  * manage its personal user account, which we will simply call "Account" instead of "Personal" (ie. a related action could be "Edit account details")
  * manage its organization users, which we will simply call "Users" instead of "Team members", and all actions related to them will refer to them as such (eg. "Add user", "Delete user")

I see two scenarios where this terminology could be confusing ([visual schema](https://app.lucidchart.com/invitations/accept/1d56541c-bcbc-4fc2-9802-2bc938e653b4)):
* For an Admin user of an API Provider Organization it could be confusing to find the same terminology used to refer to different concepts:
  * "Organization" will both refer to her own API Provider organization and to her customers' API Consumer organizations
  * Same for the term "Users"
  * I think this scenario invites us to differentiate terms:
    * one to refer to the API Provider organization, one to refer to the API Consumer organizations
    * one to refer to the API Provider organization's users, one to refer to the API Consumer organizations' users
* For an Admin who is the only user of an Organization, her personal Account and the Organization would practically mean the same thing (despite technically being two separate things)
  * In this scenario referring to the organization as something like a "Company" would recall concepts familiar also to one-people-ventures (ie. the company is a separate entity from the physical person)

To conclude, I can't seem to come up with a set of terms that would satisfy the meaning of the concepts and the contexts of use at the same time (eg. it would be strange to call an API consumer organization a company if they are internal to the same API provider company). Any feedback and/or input in this sense is very much welcome.

Reply:
@cntlsn Can you please update the mockups with this new naming arrangement so we have a clear visibility of the changes?

Reply:
@porueesq This terminology change would have consequences on pages for which mockups haven't been designed yet. Most affected pages:
**Account settings**
* Sidebar navigation
* Overview page
* Edit account details page
* Personal details page
* Users Listing / Invitations page

**Audience**
* Sidebar navigation
* Accounts listing page
* Create new account page
* Account dashboard page
* Edit account details page
* Account applications list page

Reply:
Marking this conversation as resolved, although there might be future changes due to ongoing discussions about UI terminology.